debezium-core/src/main/java/io/debezium/relational/TableSchema.java

```diff
@@ -55,23 +56,33 @@
     /**
      * Create an instance with the specified {@link Schema}s for the keys and values, and the functions that generate the
      * key and value for a given row of data.
-     * 
+     *
+     * @param parentSchemaName the name of the schema; may be null
```

Comment:
This is also unchanged from your initial draft https://github.com/debezium/debezium/pull/129/

I will think some more about naming this. I still don't 100% understand the envelope part that ends up using this, but I'll poke around more.